dune-project

```diff
@@ -15,7 +15,7 @@
  (name cppo)
  (depends
   (ocaml (>= 4.02.3))
-  (dune  (>= 1.10))
+  (dune  (>= 2.0))
```

Comment:
Maybe worth making dune a build-only dependency?
Also, should we change the depends field for `cppo_ocamlbuild` as well?

Reply:
Making `dune` a build-only dependency: this makes sense, but I don't know how to do it. In the `dune-project` file, I can write either `(dune (>= 2.0))` or `(dune :build)` but I am unable to combine these two declarations into a single one; `dune` rejects everything I try.

Reply:
Changing the `depends` field `cppo_ocamlbuild` seems preferable; I will add a commit to do it.